app/models/spree/order_decorator.rb

```diff
@@ -0,0 +1,22 @@
+Spree::Order.class_eval do
```

Comment:
Please consider using the Module Decorator pattern. You can find an example [here](https://gist.github.com/kevinnio/ec0e0b9ea4ad26988f63f27dbaf878a4).

Reply:
Already on it, thanks

Reply:
I tried to do it this way, however I was unable to remove the payment step, all of the others were possible; so for now we'll stick with this version